spec/spec.md

```diff
@@ -80,21 +80,24 @@ DELETE <ECHO_SERVER_URL>/clients/<CLIENT_ID>
 
 ### Send Notification
 
-> **Note**
-> This will have E2E encryption in the future but for now is designed without encryption.
-
 ```
 POST <ECHO_SERVER_URL>/clients/<CLIENT_ID>
 {
     "id": "0000-0000-0000-0000"
-    "payload": {
-        "title": "New Request",
-        "description": "You have a signing request."
-    }
+    "payload": "<topic>:<flags>:<blob>"
```

Comment:
this now requires us to ensure that `topic`/`flags`/`blob` don't contain any `:`.

IMO that's an unnecessary constraint and may make debugging later on kinda hard in case there is a `:` in any of those.

Reply:
Would you rather this is an object?

Reply:
Yes i think we save us some potential trouble by making it an object.

Reply:
Okay, I'll update to make it an object, it was a string as @Rakowskiii said that was something agreed